dev-libs/soci/soci-4.0.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Simple Open (Database) Call Interface - write SQL in C++"
+HOMEPAGE="https://github.com/SOCI/soci"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Boost-1.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="firebird mysql odbc oracle postgres sqlite"
+
+RDEPEND="
+	>=dev-libs/boost-1.33.1:=
+	firebird? ( dev-db/firebird )
+	mysql? ( dev-db/mysql )
+	odbc? ( dev-db/unixODBC )
+	oracle? ( dev-db/oracle-instantclient )
+	postgres? ( dev-db/postgresql:* )
+	sqlite? ( dev-db/sqlite )
```

Comment:
These USE flags have no function in this ebuild, and are banned by QA policy.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

Reply:
Ah hah, well noted!

So how do I handle, e.g. `rstudio` depending on this AND `postgresql`.....nvm just answered my own question

Reply:
@juippis it seems like `firebird` is masked, and has a bunch of masked  dependencies as well.

Would it be appropriate to place at least this one behind a `USE`-flag so that a user can decide whether or not to unmask all that stuff?

Reply:
Masked by...? At least the `firebird::gentoo` looks fine to me, no weird deps and it's only stable.masked, so wouldn't stop this at all.

Just use `optfeature`.

Reply:
Yea, I meant "masked by unstable keyword".

I'll do as you've suggested and use `optfeature`.

Reply:
This is fixed now.

Reply:
@juippis i think optfeature is the wrong tool here - unless the user is installing a binpkg, they won't see any message due to the RDEPEND.

I think I'll use einfo instead